spec/validators/fee/misc_fee_validator_spec.rb

```diff
@@ -45,6 +45,36 @@
       end
     end
 
+    context 'when misc fee is an evidence provision fee' do
+      let(:fee_type) { build :misc_fee_type, :mievi }
+
+      before { allow(fee).to receive(:fee_type).and_return(fee_type) }
+
+      context 'when the amount is set to' do
+        before { allow(fee).to receive(:amount).and_return(amount) }
+
+        %w[45 90].each do |value|
+          context "£#{value}" do
+            let(:amount) { value }
+
+            it { expect(fee).to be_valid }
+          end
+        end
+
+        context 'a decimal' do
+          let(:amount) { '45.10' }
+
+          it { should_error_if_equal_to_value(fee, :amount, amount, 'incorrect_epf') }
```

Comment:
just a suggestion, for consistency more than anything, you could get rid of the `context` and `let` and just do

```
it 'should error for decimals' do
  should_error_if_equal_to_value(fee, :amount, '45.10', 'incorrect_epf')
end
```